src/actions.ml

```diff
@@ -115,16 +115,22 @@ let send_url ~bot_info (gh_owner, gh_repo) job_info github_repo_full_name
             ~state:"success" ~url ~context ~description:(description_base ^ ".")
             ~bot_info
       | INSTALL_TOKEN _t -> (
-          GitHub_queries.get_repository_id ~bot_info ~owner:gh_owner
-            ~repo:gh_repo
-          >>= function
-          | Ok repo_id ->
-              GitHub_mutations.create_check_run ~bot_info ~name:context
-                ~status:COMPLETED ~repo_id ~head_sha:job_info.commit
-                ~details_url:url ~conclusion:SUCCESS
-                ~title:(description_base ^ ".") ~text:"" ~summary:""
-          | Error e ->
-              Lwt_io.printf "No repo id: %s" e )
+          if as_app then
+            GitHub_mutations.send_status_check
```

Comment:
Not sure I understand the point of this argument `as_app` but more importantly why this name?

Reply:
The name is poorly chosen, I updated it.